packages/frontend/src/pages/SignIn.tsx

```diff
@@ -1,7 +1,93 @@
-import React from "react";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import * as Yup from "yup";
+import Swal from 'sweetalert2'
+import withReactContent from 'sweetalert2-react-content'
+
+const MySwal = withReactContent(Swal)
+
+const validationSchema = Yup.object().shape({
+  email: Yup.string().email("Email inválido").required("Email é obrigatório"),
+  password: Yup.string()
+    .required("Senha é obrigatória")
+    .required("Senha é obrigatória"),

```

Comment:
the required rule is duplicated, instead of 2 required rules, add a min rule to avoid user to sign in with a password lesser than 8 characters